package.json

```diff
@@ -97,7 +96,8 @@
     "protractor-retry": "1.1.9",
     "selenium-webdriver": "4.0.0-alpha.1",
     "snyk": "1.70.2",
-    "wcag": "0.3.0"
+    "wcag": "0.3.0",
+    "yargs": "11.0.0"
```

Comment:
Note I had to add this because `yargs` was removed from package-lock when browser-sync is removed, but it's needed by protractor. Not sure why protractor doesn't install it, but adding it here fixes it.